flink-runtime/src/main/java/org/apache/flink/runtime/checkpoint/savepoint/SavepointV1Serializer.java

```diff
@@ -290,6 +290,7 @@ public static void serializeStreamStateHandle(StreamStateHandle stateHandle, Dat
 		} else if (stateHandle instanceof ByteStreamStateHandle) {
 			dos.writeByte(BYTE_STREAM_STATE_HANDLE);
 			ByteStreamStateHandle byteStreamStateHandle = (ByteStreamStateHandle) stateHandle;
+			dos.writeUTF(byteStreamStateHandle.getHandleName());
```

Comment:
I think it's OK that we don't change the format version (e.g. to v2) when we change the serialization format in a snapshot version right? Since 1.1 the version was already updated, which is fine.